firefox_cloud_services_PrivacyNotice/cs.md

```diff
@@ -3,26 +3,23 @@
 úterý 16. srpna 2016.
 {: datetime="2016-08-16" }
 
-Staráme se o ochranu vašeho soukromí. Když Firefox Cloud Services (dále jen „Služby”) zašle informace společnosti Mozilla (tedy nám), naše [Zásady ochrany osobních údajů Mozilla](https://www.mozilla.org/privacy/) popisují, jak můžeme takové informace používat.
+Staráme se o ochranu vašeho soukromí. Když Firefox Cloud Services (dále jen „Služby”) zašlou informace společnosti Mozilla (tedy nám), naše [Zásady ochrany osobních údajů](https://www.mozilla.org/privacy/) popisují, jak takové informace používáme.
```

Comment:
Celá tahle formulace mi přijde dost nešťastná. „Když něco zašle info, tak naše zásady budou popisovat, jak to info používáme.“, když něco info nezašle, tak se to z těch zásad vytratí? :D

Reply:
Of course for data, that are not sent, there is nothing to do. But I have reformulated the sentence in another commit.